### PR TITLE
solved the error Contributor Code of Conduct - Link - Error #148 :

### PR DESCRIPTION
**Pull request by Shu12388y**

### CODE REVIEW

**Review by BeginArjun — COMMENTED** [review]
I think you should use this link instead https://github.com/opensourcediversity/opensourcediversity.org#-contribute-code-or-design.

File: index.html [hunk 1]
```diff
@@ -445,7 +445,7 @@ <h3>Programming Diversity</h3>
 		<footer>
 			<div class="container text-center">
 				<p>Spread the word about Open Source Diversity and <a href="https://github.com/opensourcediversity" target="_blank" rel="noopener">contribute</a>!</p>
-				<p>Please note that Open Source Diversity has a <a href="https://github.com/opensourcediversity/opensourcediversity.org/blob/master/code-of-conduct.md" target="_blank" rel="noopener">Contributor Code of Conduct</a> based on the <a href="https://www.contributor-covenant.org" target="_blank" rel="noopener">Contributor Covenant</a>.<br>By participating in this project online or at events you agree to abide by its terms.</p>
+				<p>Please note that Open Source Diversity has a <a href="https://github.com/opensourcediversity/opensourcediversity.org" target="_blank" rel="noopener">Contributor Code of Conduct</a> based on the <a href="https://www.contributor-covenant.org" target="_blank" rel="noopener">Contributor Covenant</a>.<br>By participating in this project online or at events you agree to abide by its terms.</p>
```
Comment (Roshanjossey) [inline index.html:448]:
```suggestion
				<p>Please note that Open Source Diversity has a <a href="https://github.com/opensourcediversity/opensourcediversity.org/blob/main/.github/CODE_OF_CONDUCT.md" target="_blank" rel="noopener">Contributor Code of Conduct</a> based on the <a href="https://www.contributor-covenant.org" target="_blank" rel="noopener">Contributor Covenant</a>.<br>By participating in this project online or at events you agree to abide by its terms.</p>
```

**Review by Roshanjossey — CHANGES_REQUESTED** [review]
When the code of conduct was updated it was moved to here https://github.com/opensourcediversity/opensourcediversity.org/blob/main/.github/CODE_OF_CONDUCT.md

Also, could you please change it in Readme.md too? 
https://github.com/opensourcediversity/opensourcediversity.org/blame/main/README.md#L66

**Review by Roshanjossey — APPROVED** [review]

**Review by jancborchardt — APPROVED** [review]
Looks good, thanks a lot! :)

**Pull request merged**